appuio/haproxy/values.yaml

```diff
@@ -37,7 +37,7 @@ service:
   additionalPorts: []
 
 haproxy:
-  config: ldap-tls
+  config: galerak8s
```

Comment:
don't change the default config here. Leave it at `ldap-tls`

Reply:
restored